test_codecept/e2e/features/step_definitions/searchCase.steps.js

```diff
@@ -174,6 +174,14 @@ const RuntimeTestData = require("../../support/runtimeTestData");
 
   });
 
+  When('I enter case reference number {string}', async function (caseRefNumber) {
```

Comment:
to avoid confusion, can we move NoC specific functions  to its own step definitions file